2020/07.md

```diff
@@ -89,6 +89,7 @@ Supporting materials includes slides, a link to the proposal repository, a link
 |     |   3   |   15m   | [`Promise.any` & `AggregateError`](https://github.com/tc39/proposal-promise-any) for stage 4 ([slides](https://docs.google.com/presentation/d/1PcfPP5nLVG8w5OosQt5PWwOmFUPuD5pg1rgGaF1JVis/edit)) ([PR](https://github.com/tc39/ecma262/pull/2040)) | Mathias Bynens |
 |     |   3   |   15m   | [WeakRefs](https://github.com/tc39/proposal-weakrefs/) for Stage 4 ([PR](https://github.com/tc39/ecma262/pull/2089)) and [CleanupSome](https://github.com/codehag/proposal-cleanup-some) for stage 3 | Daniel Ehrenberg, Yulia Startsev |
 |     |   3   |   15m   | [Logical Assignment](https://github.com/tc39/proposal-logical-assignment/) for Stage 4 ([PR](https://github.com/tc39/ecma262/pull/2030), [slides](https://docs.google.com/presentation/d/1rq448C-mzxWzBs6RTInczGTjs4xv5gQenDJY1fGogX0/edit?usp=sharing)) | Justin Ridgewell |
+|     |   2   |   10m   | [Decorators](https://github.com/tc39/proposal-decorators/) status update ([slides](https://slides.com/pzuraq/decorators-3cb407/edit)) | Chris Hewell Garrett |
```

Comment:
The slides link asks for a log in - are the slides public?

Reply:
it works without the `/edit` at the end of the link
https://github.com/tc39/agendas/pull/832

Reply:
it works without the `/edit` at the end of the link
PR with fix https://github.com/tc39/agendas/pull/832